packages/ses/src/lockdown.js

```diff
@@ -180,12 +181,20 @@ export const repairIntrinsics = (options = {}) => {
       /** @param {string} debugName */
       debugName => debugName !== '',
     ),
+    regeneratorRuntimeTaming = getenv(
+      'LOCKDOWN_REGENERATOR_RUNTIME_TAMING',
+      'none',
+    ),
     __hardenTaming__ = getenv('LOCKDOWN_HARDEN_TAMING', 'safe'),
     dateTaming = 'safe', // deprecated
     mathTaming = 'safe', // deprecated
     ...extraOptions
   } = options;
 
+  regeneratorRuntimeTaming === 'safe' ||
```

Comment:
The behavior this flag causes is ***not*** safe, since it causes assignments to fail silently, allowing control flow to proceed into code that assumed the assignment succeeded. When we introduced strict mode in ES5, that's why we made failed assignments throw. Because the ES3 sloppy behavior was ***unsafe***.

I understand that silently ignoring the particular problematic assignment done by legacy regenerator runtime is safe, but I don't see any good way to make the behavior change without the hazard of allowing other innocent assignments to also fail silently.

I suggest renaming the `'none'` option to `'safe'` and renaming this `'safe'` option to `'unsafe'` or `'unsafe-something'`.